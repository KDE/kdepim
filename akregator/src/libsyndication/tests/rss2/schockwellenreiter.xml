<?xml version="1.0" encoding="iso-8859-1"?><rss version="0.92"><!-- Source: www.schockwellenreiter.de     titles: plain/text in CDATA     description: HTML in CDATA    --><channel>	<title>Der Schockwellenreiter (RSS-Feed)</title>	<link>http://www.schockwellenreiter.de/</link>	<description>Die taegliche Ration Wahnsinn</description>	<language>de-de</language>	<copyright>Copyright 2000 - 2006</copyright>	<image>		<url>http://www.schockwellenreiter.de/mathe/images/me2.jpeg</url>		<title>Der Schockwellenreiter</title>		<link>http://www.schockwellenreiter.de/impressum.html</link>		</image>	<item>	<title><![CDATA[Päuschen (auf der Rixdorfer Bühne)]]></title>	<description><![CDATA[Morgen muß ich ‹ für meine Verhältnisse ‹ sehr früh raus und es wird daher vermutlich morgen kein Update geben. Das Hundebild am Freitag gibt es so diese Woche schon am Donnerstag abend.		<a href="http://www.flickr.com/photos/schockwellenreiter/113422989/" title="Rixdorfer Bühne, Wandgemälde (mit Hund) der Künstlergruppe Ratgeb in der Richardstraße"><img src="http://www.schockwellenreiter.de/images/rixdorf_0011.jpg" width="460" height="345" border="0" alt="Rixdorfer Bühne, Wandgemälde in der Richardstraße, Photo: Gabriele Kantel, 16.03.2006" /></a>		Und am Wochenende findet wieder (hoffentlich ‹ wenn das Wetter mitspielt) <a href="http://www.hsvtempelhof.de/"><b>Hundeschule</b></a> und <a href="http://schockwellenreiter.server-wg.de/schockwellenreiter/texte/agility/"><b>Agility</b></a>-Training statt. Das heißt: Sicher ist nur, daß es Montag früh wieder neue Meldungen hier geben wird. Alles andere wird Euch (und mich) überraschen. Daher schon heute ein schönes Wochenende Euch allen da draußen und bleibt uns gewogen... <i>[Photo: <a href="http://www.kantel.de/spielplatz/gabi.html">Gabi</a>]</i>		<strong style="color:#3993ff">flick<span style="color:#ff1c92">r</span></strong>-Photoset: <a href="http://www.flickr.com/photos/schockwellenreiter/sets/72057594083629471/"><b>Rixdorfer Bühne</b></a> ­ ein Wandgemälde der Künstlergruppe Ratgeb in der Neuköllner Richardstraße.		<span class='small'>Technorati-Tags: <a href='http://technorati.com/tag/Rixdorf' rel='tag'>Rixdorf</a> <a href='http://technorati.com/tag/Ratgeb' rel='tag'>Ratgeb</a> </span>		]]></description>		<link>http://www.schockwellenreiter.de/2006/03/16.html#puschenaufDerRixdorferBhne</link>	</item>	</channel></rss>