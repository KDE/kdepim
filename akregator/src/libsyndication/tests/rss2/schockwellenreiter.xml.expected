# Feed begin ######################
title: #Der Schockwellenreiter (RSS-Feed)#
link: #http://www.schockwellenreiter.de/#
description: #Die taegliche Ration Wahnsinn#
copyright: #Copyright 2000 - 2006#
language: #de-de#
# Image begin #####################
url: #http://www.schockwellenreiter.de/mathe/images/me2.jpeg#
title: #Der Schockwellenreiter#
link: #http://www.schockwellenreiter.de/impressum.html#
height: #31#
width: #88#
# Image end #######################
# Item begin ######################
id: #hash:828401ddef7d81051adf7dca29b085d2#
title: #Päuschen (auf der Rixdorfer Bühne)#
link: #http://www.schockwellenreiter.de/2006/03/16.html#puschenaufDerRixdorferBhne#
description: #Morgen muß ich  für meine Verhältnisse  sehr früh raus und es wird daher vermutlich morgen kein Update geben. Das Hundebild am Freitag gibt es so diese Woche schon am Donnerstag abend. <a href="http://www.flickr.com/photos/schockwellenreiter/113422989/" title="Rixdorfer Bühne, Wandgemälde (mit Hund) der Künstlergruppe Ratgeb in der Richardstraße"><img src="http://www.schockwellenreiter.de/images/rixdorf_0011.jpg" width="460" height="345" border="0" alt="Rixdorfer Bühne, Wandgemälde in der Richardstraße, Photo: Gabriele Kantel, 16.03.2006" /></a> Und am Wochenende findet wieder (hoffentlich  wenn das Wetter mitspielt) <a href="http://www.hsvtempelhof.de/"><b>Hundeschule</b></a> und <a href="http://schockwellenreiter.server-wg.de/schockwellenreiter/texte/agility/"><b>Agility</b></a>-Training statt. Das heißt: Sicher ist nur, daß es Montag früh wieder neue Meldungen hier geben wird. Alles andere wird Euch (und mich) überraschen. Daher schon heute ein schönes Wochenende Euch allen da draußen und bleibt uns gewogen... <i>[Photo: <a href="http://www.kantel.de/spielplatz/gabi.html">Gabi</a>]</i> <strong style="color:#3993ff">flick<span style="color:#ff1c92">r</span></strong>-Photoset: <a href="http://www.flickr.com/photos/schockwellenreiter/sets/72057594083629471/"><b>Rixdorfer Bühne</b></a> ­ ein Wandgemälde der Künstlergruppe Ratgeb in der Neuköllner Richardstraße. <span class='small'>Technorati-Tags: <a href='http://technorati.com/tag/Rixdorf' rel='tag'>Rixdorf</a> <a href='http://technorati.com/tag/Ratgeb' rel='tag'>Ratgeb</a> </span>#
# Item end ########################
# Feed end ########################

